yw_website/yw_rest_services/views.py

```diff
@@ -61,8 +61,38 @@ def create_workflow(request):
         'run': rdata
         })
 
-    
- 
-
-
+@api_view(['post'])
+@permission_classes((permissions.AllowAny,))
+def update_workflow(request, workflow_id):
+    try:
+        w = Workflow.objects.get(pk=workflow_id)
+    except Workflow.DoesNotExist:
+        return Response(status=500, data={'error':'workflow does not exist'})
 
+    v = Version(
```

Comment:
my main feedback for this commit:
* if `model_checksum` does not exist in `request.data`, we want to return a `500 Bad Request`
* check the `model_checksum` value of all versions of this workflow. If the checksum value doesn't match with any of the versions' checksums, then we create a new version. Otherwise, we use the version that has the same checksum as in the request.
i.e something that looks a bit like this
```
v = Version.objects.filter(workflow=workflow_id).filter(model_checksum=request.data.get('model_checksum', None)
if v is None:
    # create new version
```

Disclaimer, i don't know if this exact code works